sdks/python/README.md

```diff
@@ -1 +1,80 @@
-Please see updated documentation [here](https://docs.omi.me/docs/developer/sdk/python)
+<!-- This file is auto-generated from docs/docs/docs/developer/sdk/python.mdx. Do not edit manually. -->
+# 🎧 Omi Python SDK 
+
+This SDK connects to an **Omi wearable device** over **Bluetooth**, decodes the **Opus-encoded audio**, and transcribes it in **real time using Deepgram**.
```

Comment:
how did you autogenerate this readme? I believe it's not relate to this PR as well. please check this others and below